brayns/engine/components/LoadInformation.h

```diff
@@ -18,24 +18,26 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include "MaterialComponent.h"
+#pragma once
 
-#include <brayns/engine/material/types/Phong.h>
+#include <brayns/json/JsonType.h>
+
+#include <string>
 
 namespace brayns
 {
-MaterialComponent::MaterialComponent()
-    : _material(Phong())
+struct LoadInformation
 {
-}
+    enum class LoadType
```

Comment:
Just personal preference but I would call that LoadSource {Unknown, File, Blob} and put it outside the class so it is Unknown if not set (default value) and we don't need to do LoadInformation::LoadType to access it.

Reply:
Addressed